src/RenderedHelper.ts

```diff
@@ -23,6 +24,17 @@ export class RenderedHelper {
         this._updateRenderedPdf(response);
     }
 
+    public static createHttpRequestContent(httpConfig: HttpStepConfiguration): string {
+        let headers = '';
+        if (httpConfig.headers) {
+            headers = Object.keys(httpConfig.headers).map(key => `${key}: ${httpConfig.headers?.[key]}`).join('\r\n');
```

Comment:
Maybe we should look into using `os.EOL` later, instead of using Windows specific new line characters

Reply:
Good point, will pickup in later release!